test/standard/standard.test.js

```diff
@@ -1,13 +0,0 @@
-const { run } = require('../utils/test-utils');
-
-describe('standard output', () => {
-    it('should print standard output', () => {
-        const { stdout, stderr } = run(__dirname);
```

Comment:
I think this test is not for `--standard` flag but the default webpack output (referred to as standard). Please correct me if I'm wrong.

Reply:
No, this was used for --standard, but later --standard was dropped from the test when flag was removed.

Reply:
So, do we have another test for default output or we don't need one?

Reply:
zero-config

Reply:
I've added the things missing there, from this test we removed.